### ISSUE TITLE
Incorrect option_groups response

### ISSUE THREAD

**Issue by lfbrandao** [slackapi/java-slack-sdk#662]
The JSON response to block suggestion requests seems incorrect when returning `option_groups`.  More specifically, the `label` property is being returned as a string instead of an object with `type` and `text` properties.

I'm attaching some code below so that you can reproduce the issue. Let me know if there's something wrong on my end. Thanks!

#### The Slack SDK version

```
+--- com.slack.api:bolt-jetty:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0
|    +--- com.slack.api:slack-api-client:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    +--- com.slack.api:slack-app-backend:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    +--- com.slack.api:bolt:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    \--- com.slack.api:slack-app-backend:1.5.0 (*)
|    +--- com.slack.api:bolt-servlet:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    +--- com.slack.api:slack-app-backend:1.5.0 (*)
|    |    \--- com.slack.api:bolt:1.5.0 (*)
+--- com.slack.api:slack-api-model-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0 (*)
+--- com.slack.api:slack-api-client-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model-kotlin-extension:1.5.0 (*)
|    +--- com.slack.api:slack-api-client:1.5.0 (*)
+--- com.slack.api:bolt-jetty:1.5.0 (n)
+--- com.slack.api:slack-api-model-kotlin-extension:1.5.0 (n)
+--- com.slack.api:slack-api-client-kotlin-extension:1.5.0 (n)
+--- com.slack.api:bolt-jetty:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0
|    +--- com.slack.api:slack-api-client:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    +--- com.slack.api:slack-app-backend:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    +--- com.slack.api:bolt:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    \--- com.slack.api:slack-app-backend:1.5.0 (*)
|    +--- com.slack.api:bolt-servlet:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    +--- com.slack.api:slack-app-backend:1.5.0 (*)
|    |    \--- com.slack.api:bolt:1.5.0 (*)
+--- com.slack.api:slack-api-model-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0 (*)
+--- com.slack.api:slack-api-client-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model-kotlin-extension:1.5.0 (*)
|    +--- com.slack.api:slack-api-client:1.5.0 (*)
+--- com.slack.api:bolt-jetty:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0
|    +--- com.slack.api:slack-api-client:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    +--- com.slack.api:slack-app-backend:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    +--- com.slack.api:bolt:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    \--- com.slack.api:slack-app-backend:1.5.0 (*)
|    +--- com.slack.api:bolt-servlet:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    +--- com.slack.api:slack-app-backend:1.5.0 (*)
|    |    \--- com.slack.api:bolt:1.5.0 (*)
+--- com.slack.api:slack-api-model-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0 (*)
+--- com.slack.api:slack-api-client-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model-kotlin-extension:1.5.0 (*)
|    +--- com.slack.api:slack-api-client:1.5.0 (*)
+--- com.slack.api:bolt-jetty:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0
|    +--- com.slack.api:slack-api-client:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    +--- com.slack.api:slack-app-backend:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    +--- com.slack.api:bolt:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    \--- com.slack.api:slack-app-backend:1.5.0 (*)
|    +--- com.slack.api:bolt-servlet:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    +--- com.slack.api:slack-app-backend:1.5.0 (*)
|    |    \--- com.slack.api:bolt:1.5.0 (*)
+--- com.slack.api:slack-api-model-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0 (*)
+--- com.slack.api:slack-api-client-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model-kotlin-extension:1.5.0 (*)
|    +--- com.slack.api:slack-api-client:1.5.0 (*)
+--- com.slack.api:bolt-jetty:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0
|    +--- com.slack.api:slack-api-client:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    +--- com.slack.api:slack-app-backend:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    +--- com.slack.api:bolt:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    \--- com.slack.api:slack-app-backend:1.5.0 (*)
|    +--- com.slack.api:bolt-servlet:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    +--- com.slack.api:slack-app-backend:1.5.0 (*)
|    |    \--- com.slack.api:bolt:1.5.0 (*)
+--- com.slack.api:slack-api-model-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0 (*)
+--- com.slack.api:slack-api-client-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model-kotlin-extension:1.5.0 (*)
|    +--- com.slack.api:slack-api-client:1.5.0 (*)
+--- com.slack.api:bolt-jetty:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0
|    +--- com.slack.api:slack-api-client:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    +--- com.slack.api:slack-app-backend:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    +--- com.slack.api:bolt:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    \--- com.slack.api:slack-app-backend:1.5.0 (*)
|    +--- com.slack.api:bolt-servlet:1.5.0
|    |    +--- com.slack.api:slack-api-model:1.5.0 (*)
|    |    +--- com.slack.api:slack-api-client:1.5.0 (*)
|    |    +--- com.slack.api:slack-app-backend:1.5.0 (*)
|    |    \--- com.slack.api:bolt:1.5.0 (*)
+--- com.slack.api:slack-api-model-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model:1.5.0 (*)
+--- com.slack.api:slack-api-client-kotlin-extension:1.5.0
|    +--- com.slack.api:slack-api-model-kotlin-extension:1.5.0 (*)
|    +--- com.slack.api:slack-api-client:1.5.0 (*)
```

#### Java Runtime version

```
openjdk version "14.0.2" 2020-07-14
OpenJDK Runtime Environment (build 14.0.2+12-46)
OpenJDK 64-Bit Server VM (build 14.0.2+12-46, mixed mode, sharing)
```

#### OS info

```
ProductName:	macOS
ProductVersion:	11.0.1
BuildVersion:	20B29
Darwin Kernel Version 20.1.0: Sat Oct 31 00:07:11 PDT 2020; root:xnu-7195.50.7~2/RELEASE_X86_64
```

#### Steps to reproduce:

1. Run [this app](https://github.com/lfbrandao/slack-state-debug) and configure a Slack app accordingly
2. Execute the `/open-modal` command on Slack and open the select menu
3. The select menu won't show any results

#### Expected result:

I expected the select menu to load the results

#### Actual result:

The app linked above responds with [this](https://github.com/lfbrandao/slack-state-debug/blob/master/incorrect-option-groups.json), but I expected the response to be like [this](https://github.com/lfbrandao/slack-state-debug/blob/master/expected-option-groups.json)

### Requirements

Please make sure if this topic is specific to this SDK. For general questions/issues about Slack API platform or its server-side, could you submit questions at https://my.slack.com/help/requests/new instead. :bow:

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to the those rules.

**Comment by seratch** [comment]
@lfbrandao Thanks a lot for taking the time to report this with a working example (thanks, it's really helpful!). And, you are right. This has been a bug for a while... I will fix this and release a new patch version shortly.

**Comment by seratch** [comment]
@lfbrandao Thanks again for reporting this. I just published v1.5.1 to the Sonatype OSS repositories. The jar files will be available on Maven Central soon.

**Comment by lfbrandao** [comment]
@seratch it's working now! thanks so much for the quick turnaround.
